packages/infra/dashboards/cw-dashboard-prod.json

```diff
@@ -389,8 +389,8 @@
                 "annotations": {
                     "horizontal": [
                         {
-                            "label": "VolumeWriteIOPs >= 100_000 for 1 datapoints within 5 minutes",
-                            "value": 100000,
+                            "label": "VolumeWriteIOPs >= 5000 for 1 datapoints within 5 minutes",
```

Comment:
Why are we changing this back to 5000? I thought we updated the threshold to 100k

Reply:
I didn't update this section of the dashboard, was it updated to 100K when we updated CDK/infra?

Reply:
oh, we likely updated the source of the dashboard but didn't update the dashboard on AWS... I'll try to fix this